ci-operator/step-registry/openshift/e2e/test/pausedworkers/openshift-e2e-test-pausedworkers-ref.yaml

```diff
@@ -0,0 +1,64 @@
+ref:
+  as: openshift-e2e-test-pausedworkers
+  from: tests
+  commands: openshift-e2e-test-pausedworkers-commands.sh
+  timeout: 10800s
+  env:
+  - name: TEST_TYPE
+    default: "suite"
+    documentation: |
+      The type of test to perform.
+
+      * 'suite' - Run the test suite defined by TEST_SUITE. By default this executes the
+                  OpenShift parallel conformance suite. All product components must run the
+                  default suite on pull requests prior to merge.
+      * 'suite-conformance' - Runs the test suite defined by TEST_SUITE, then runs the product conformance
+                  suite to verify the cluster is still functional. This is typically used for testing
+                  disruption from one suite, then confirming that the disruption did not degrade the
+                  product.
+      * 'upgrade' - Perform an upgrade to the image defined by OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE.
+                  The TEST_UPGRADE_OPTIONS flag may be optionally set. All product components
+                  must run the default upgrade test (verify an upgrade completes) on pull requests
+                  prior to merge.
+      * 'upgrade-conformance' - Performs the 'upgrade' test and then executes the full conformance
+                  suite after upgrade completes. Does not honor TEST_SUITE but will respect
+                  TEST_UPGRADE_OPTIONS. All product releases must pass the conformance suite after
+                  an upgrade completes or have explicit logic in their test to tolerate behavior after
+                  upgrade.
+  - name: TEST_SUITE
+    default: openshift/conformance/parallel
+    documentation: The test suite to run.  Use 'openshift-test run --help' to list available suites.
+  - name: TEST_SKIPS
+    default: ""
+    documentation: Regular expression (POSIX basic regular expression) of tests to skip.
+  - name: TEST_UPGRADE_OPTIONS
+    default: ""
+    documentation: Options controlling how an upgrade is performed. See `openshift-test run-upgrade --help` for more details.
+  - name: TEST_REQUIRES_SSH
+    default: ""
+    documentation:
+      If the test suite requires SSH access, setting this variable will install a bastion and configure
+      the Kube e2e test infrastructure to use the bastion. Otherwise, tests that require SSH will fail because
+      OpenShift does not expose SSH to nodes by default.
+  - name: TEST_INSTALL_CSI_DRIVERS
+    default: ""
+    documentation: The CSI driver(s) to use.  Used by the `openshift-tests` program, see its documentation for details.
+  - name: TEST_CSI_DRIVER_MANIFEST
+    default: ""
+    documentation: |-
+      Name of the CSI driver manifest file to use.  Used by the `openshift-tests`
+      program as TEST_CSI_DRIVER_FILES env. var., see its documentation for
+      details. The file must be present in ${SHARED_DIR}.
+  dependencies:
```

Comment:
This step introduces new dependencies, so I can't merge it back to `openshift-e2e-test`